tests/test_standard_tags.py

```diff
@@ -41,7 +41,9 @@ def test_method_trace(self):
         interfaces.library.add_span_tag_validation(request=r, tags=tags)
 
 
-@released(dotnet="2.13.0", golang="1.40.0", java="0.107.1", nodejs="3.0.0", php="0.76.0", python="?", ruby="?")
+@released(
+    dotnet="2.13.0", golang="1.40.0", java="0.107.1", nodejs="3.0.0", php="0.76.0", python="1.6.0rc1.dev", ruby="?"
```

Comment:
current dev version is  `python@1.5.0rc2.dev48+gd795cbeb` : https://github.com/DataDog/system-tests/actions/runs/3241192285/jobs/5312822840#step:9:43

it means that may not be passing, and it also means that when you'll update your repo, it may break the CI.

Reply:
More details about that : to install the python "dev" tracer, we do this : 

```bash
pip install git+https://github.com/DataDog/dd-trace-py.git
```

Then we do this to get the version

```bash
python -c "import ddtrace; print(ddtrace.__version__)"
```

Turns out  the actual system of dd-trace-py versionning does not take in account the last released version, because it's `1.5.2` ...